theano/gpuarray/tests/test_type.py

```diff
@@ -79,8 +79,9 @@ def test_unpickle_gpuarray_as_numpy_ndarray_flag0():
     finally:
         config.experimental.unpickle_gpu_on_cpu = oldflag
 
-
-test_shared_options = makeSharedTester(
+# These tests are disabled because they expect the impossible
```

Comment:
Was this tested in the old back-end? I think you took that from there. So this is a regression compared to the old back-end.

I don't want to delay this PR for this, can you make an issue about this and tag it for 0.10 and CCW so it don't get delayd too much? Maybe Reyhane could do that too.